tox.ini

```diff
@@ -66,10 +66,10 @@ deps =
     # dependencies that have a minimum version.
     # pytest-openfiles pinned because of https://github.com/astropy/astropy/issues/10160
     oldestdeps: numpy==1.17.*
-    oldestdeps: matplotlib==2.1.*
+    oldestdeps: matplotlib==3.0.*
     oldestdeps: asdf==2.6.*
-    oldestdeps: scipy==0.18.*
-    oldestdeps: pyyaml==3.12
+    oldestdeps: scipy==1.1.*
```

Comment:
We list scipy>1.0 above, so should this be `1.0.*`?

Reply:
yes, these all should be updated, I would do it once the concept of bumping stuff gets the green light as per https://github.com/astropy/astropy/pull/10900#issuecomment-715641100

Reply:
Makes sense; green light from me!